serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc

```diff
@@ -0,0 +1,359 @@
+= Timeouts on events for {context}
+:compat-mode!:
+// Metadata:
+:description: Using timeouts in {context}
+:keywords: kogito, workflow, serverless, timeout, timer, expiration
+
+When defining a state in a workflow you can set a timeout which is a property used to control the maximum time this state should take to be complete. In case of the time is elapsed and overdue, this state is considered timed-out, and the workflow moves to the next stated defined in the transition. The properties that you can use when defining a state timeout are described in the link:{spec_doc_url}#event-timeout-definition[Serverless Workflow specification].
+
+The event-based states can define a property `eventTimeout` which uses ISO 8601 data and time format to specify a duration of time, for instance, you can if you can represent 15 minutes like `PT15M`,  to represent 15 minutes or 2 days, 3 hours and 4 minutes `P2DT3H4M`.
```

Comment:
I believe that here we can maybe include something like this:

"Event-based states can use the sub-property eventTimeout to configure the maximum time to wait for an event to arrive."

Since "timeouts" is a composed property. Another alternative could be to start saying in the initial paragrap that timeouts is a composed thing, but I think it's ok as it is, since we introduce the concept.

Reply:
thanks

Reply:
wait!

I believe this piece needs some more love:

"for instance, you can if you can represent 15 minutes like PT15M, to represent 15 minutes or 2 days, 3 hours and 4 minutes P2DT3H4M."

Reply:
+1, changing the text